docs/RULES.md

```diff
@@ -372,7 +372,7 @@ Tags: line_length
 
 Aliases: line-length
 
-Parameters: line_length, code_blocks, tables (number; default 80, boolean; default true, boolean; default true)
+Parameters: line_length, code_blocks, ignore_code_blocks, tables (number; default 80, boolean; default true, boolean; default false, boolean; default true)
```

Comment:
lets put our preferred one first.

Reply:
I changed the order here and also in the `RULES.md` such that the documentation test (which checks the order of the parameters) succeeds. 

Do we have a "standard" approach to warn of deprecated parameters? I would like to print some warning on the screen when the `code_blocks` parameter is used in order to raise awareness of the deprecation. Should this go in a `warn` statement or would we normally use a `puts`/`print` statement?